migration_steps/validation/validate_db/app/app.py

```diff
@@ -96,11 +69,178 @@ def main(verbose):
     print(tabulate(report_data, headers, tablefmt="psql"))
 
 
-    log.info("Migration Validation")
-    conn_migration = psycopg2.connect(config.get_db_connection_string("migration"))
-    conn_target = psycopg2.connect(config.get_db_connection_string("target"))
+def get_sirius_table(mapping):
+    mapping_name_to_table = {
+        "client_persons": "persons",
+        "client_addresses": "addresses",
+        "client_phonenumbers": "phonenumbers"
+    }
+    return mapping_name_to_table.get(mapping, mapping)
+
+
+def get_exception_table(mapping):
+    return f"casrec_migration_exceptions_{mapping}"
+
+
+def get_map_dict(mapping):
```

Comment:
Very similar method in `migration_steps/shared/helpers.py` - could you use/modify the shared verison instead?

Reply:
Done :)